t/t2211-job-manager-events.t

```diff
@@ -0,0 +1,181 @@
+#!/bin/sh
+
+test_description='Test flux job manager events service'
+
+. `dirname $0`/job-manager/job-manager-helper.sh
+
+. $(dirname $0)/sharness.sh
+
+test_under_flux 4
+
+RPC=${FLUX_BUILD_DIR}/t/request/rpc
+EVENT_STREAM=${FLUX_BUILD_DIR}/t/job-manager/event_stream
```

Comment:
I think you could just use `t/request/rpc_stream` here instead of adding a new C program for these tests.  It's used by  `t2310-resource-module.t` and `t0025-broker-state-machine.t` to test test their streaming RPCs (for example).  Feel free to add options if you need them.

Or - a  python one liner that uses the Flux API is a very low-complexity way to do something one-off in a test like this. @grondo prodded me to use one in `t0001-basic.t` e.g.

```
test_expect_success 'flux-start with size 1 has no peers' '
        flux start ${ARGS} --size=1 \
                flux python -c "import flux; print(flux.Flux().rpc(\"overlay.lspeer\").get())" >idle.out &&
        test_must_fail grep idle idle.out
```

Reply:
> I think you could just use t/request/rpc_stream here instead of adding a new C program for these tests.

I originally did, but wanted to cover the `job-manager.events-cancel` case, which `rpc_stream` doesn't.

Let me actually update `event_stream` to output each event separately on a new line.  It'll be necessary when I convert `job-info` to use annotations.  (The `alloc` eventlog entry includes an annotation in the context, so using `grep` will no longer be useful in testing).